messages/playback/playback_file.json

```diff
@@ -0,0 +1,40 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "title": "Playback of file",
+  "description": "Playback control and actions of log file playback micro service.",
```

Comment:
It says that the content of this file is a "playback control and actions" then it should go in `messages/commands` according to me.

Reply:
Perhaps we should make a distinction between a `command` (aimed for the entity which is monitored/remotely controlled through Crowsnest) and `action` (which could be for communication inbetween microservices, for exmaple playback actions and video-feed negotiating actions etc). One thing to ponder on is the scope of Brefv and if there is something that should not go in here at all?